jobs/cluster-6/spec

```diff
@@ -4,13 +4,23 @@ packages:
   - redis-6
 templates:
   bin/ctl: bin/ctl
+  bin/pre-start: bin/pre-start
   bin/post-deploy: bin/post-deploy
   bin/monit_debugger: bin/monit_debugger
-  config/redis.conf: config/redis.conf
+
   data/properties.sh.erb: data/properties.sh
+  
   helpers/ctl_setup.sh: helpers/ctl_setup.sh
   helpers/ctl_utils.sh: helpers/ctl_utils.sh
 
+  config/tls/redis.ca: config/tls/redis.ca
```

Comment:
The design decision for a universal blacksmith services CA was made a while back. Ioannis was working on this please check with him. The Blacksmith Services CA should be used to sign the redis server cert and key.

Reply:
I already verified that redis is using certificates generated from the Blacksmith CA. @itsouvalas could still take a look, the implementation is the same as RabbitMQ.